tests/testthat/test-find_groups.R

```diff
@@ -1,32 +1,8 @@
 test_that("find_groups() success case", {
-  withr::local_options(list(meetupr.use_oauth = FALSE))
-  set_api_key("R-Ladies FTW!")
 
-  meetup_groups <- with_mock(
```

Comment:
`with_mock()` is deprecated, and anyway things are easier with vcr so I've modernised the tests.